src/renderer/easingFunctions.ts

```diff
@@ -0,0 +1,151 @@
+//https://github.com/ai/easings.net/blob/master/src/easings/easingsFunctions.ts
```

Comment:
easings.net has a GPL 3.0 license and without doing a deep research that looks line a no go for us (e.g. https://opensource.stackexchange.com/a/1641)